lms/validation/_base.py

```diff
@@ -149,7 +149,7 @@ def parse(self, *args, **kwargs):
     def _pre_load(self, response, **_kwargs):  # pylint: disable=no-self-use
         try:
             return response.json()
-        except ValueError as err:
+        except (AttributeError, ValueError) as err:
```

Comment:
`AttributeError` is what happens if the given `response` isn't a `requests.Response` object (e.g. if it's `None`)